tests/sf_service_config_ut/sf_service_config_ut_helpers.h

```diff
@@ -14,7 +14,7 @@
 #include "macro_utils/macro_utils.h"
 
 #include "c_util/rc_string.h"
-#include "c_util/thandle.h"
+#include "c_pal/thandle.h"
```

Comment:
>c_pal/thandle [](http://example.com/codeflow?start=10&length=13)

How come no deps changes in this PR?
I'd expect the deps to be updates together with switching the repo where we're getting thandle from...

Reply:
The deps change came in the PR before.  This was a miss.